packages/app/src/components/navigation/links/index.tsx

```diff
@@ -1,4 +1,5 @@
 import classnames from 'classnames';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
```

Comment:
Since import from 'react-i18n' is also possible, it would be nice to control where 'useTranslation' is imported from.

Reply:
I will control it by eslint in another PR.